mod/thewire/views/default/thewire.js

```diff
@@ -7,7 +7,7 @@ elgg.thewire.init = function() {
 	var callback = function() {
 		var maxLength = $(this).data('max-length');
 		if (maxLength) {
-			elgg.thewire.textCounter(this, $("#thewire-characters-remaining span"), maxLength);
+			elgg.thewire.textCounter(this, $(this).parents('.elgg-form-thewire-add').find("#thewire-characters-remaining span"), maxLength);
```

Comment:
ah ok... but then it could/should also apply to the .find in your next comment (`#thewire-submit-button`) am i right?